### PR TITLE
Various docfixes

### PR DESCRIPTION
**Pull request by drammock**
some low-hanging fruit that was left over from the sprint.

### CODE REVIEW

**Comment by drammock** [comment]
this one is good to go. Here's an example of the addition to the `Notes` section, rendered:
https://output.circle-artifacts.com/output/job/3bd85ef3-0338-43aa-8da4-f2856a39b567/artifacts/0/dev/generated/mne.preprocessing.ICA.html#mne.preprocessing.ICA.plot_sources

@cbrnr I think you requested that addition, care to review/merge?

File: mne/utils/docs.py [hunk 1]
```diff
@@ -1995,6 +1995,23 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
     False.
 """
 
+docdict['notes_2d_backend'] = """\
+MNE-Python provides two different backends for signal-browsing plots (i.e.,
```
Comment (cbrnr) [inline mne/utils/docs.py:1999]:
```suggestion
MNE-Python provides two different backends for browsing plots (i.e.,
```

File: mne/utils/docs.py [hunk 2]
```diff
@@ -1995,6 +1995,23 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
...
+:meth:`raw.plot()<mne.io.Raw.plot>`, :meth:`epochs.plot()<mne.Epochs.plot>`,
+and :meth:`ica.plot_sources()<mne.preprocessing.ICA.plot_sources>`). One is
+based on :mod:`matplotlib` and the other is based on
```
Comment (cbrnr) [inline mne/utils/docs.py:2002]:
```suggestion
based on :mod:`matplotlib`, and the other is based on
```

File: mne/utils/docs.py [hunk 3]
```diff
@@ -1995,6 +1995,23 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
...
+:doc:`PyQtGraph<pyqtgraph:index>`. You can set the backend for the
+duration of a Python session using :func:`mne.viz.set_browser_backend`, or you
+can change the backend temporarily with the context manager
+:func:`mne.viz.use_browser_backend`.
+
+.. note:: For the PyQtGraph backend to run in IPython with ``block=False``
+          you must run the magic command ``%%gui qt5`` first.
```
Comment (cbrnr) [inline mne/utils/docs.py:2009]:
```suggestion
          you must run the magic command ``%gui qt5`` first.
```

File: mne/utils/docs.py [hunk 4]
```diff
@@ -1995,6 +1995,23 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
...
 
+docdict['notes_2d_backend'] = """\
+MNE-Python provides two different backends for signal-browsing plots (i.e.,
+:meth:`raw.plot()<mne.io.Raw.plot>`, :meth:`epochs.plot()<mne.Epochs.plot>`,
+and :meth:`ica.plot_sources()<mne.preprocessing.ICA.plot_sources>`). One is
+based on :mod:`matplotlib` and the other is based on
+:doc:`PyQtGraph<pyqtgraph:index>`. You can set the backend for the
+duration of a Python session using :func:`mne.viz.set_browser_backend`, or you
+can change the backend temporarily with the context manager
+:func:`mne.viz.use_browser_backend`.
```
Comment (cbrnr) [inline mne/utils/docs.py:2006]: I think it would be nice if this note mentioned how to permanently set the backend.

Comment (drammock) [inline mne/utils/docs.py:2006]:
```suggestion
:doc:`PyQtGraph<pyqtgraph:index>`. You can set the backend temporarily with the 
context manager :func:`mne.viz.use_browser_backend`; you can set it for the
duration of a Python session using :func:`mne.viz.set_browser_backend`, and you
can set the default via
:func:`mne.set_config('MNE_BROWSER_BACKEND', 'matplotlib')<mne.set_config>`
(or ``'qt'``).
```

File: mne/utils/docs.py [hunk 5]
```diff
@@ -1995,6 +1995,25 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
     False.
 """
 
+docdict['notes_2d_backend'] = """\
+MNE-Python provides two different backends for browsing plots (i.e.,
+:meth:`raw.plot()<mne.io.Raw.plot>`, :meth:`epochs.plot()<mne.Epochs.plot>`,
+and :meth:`ica.plot_sources()<mne.preprocessing.ICA.plot_sources>`). One is
+based on :mod:`matplotlib`, and the other is based on
+:doc:`PyQtGraph<pyqtgraph:index>`. You can set the backend temporarily with the
```
Comment (drammock) [inline mne/utils/docs.py:2003]:
```suggestion
:doc:`PyQtGraph<pyqtgraph:index>`. You can set the backend temporarily with the
```

File: requirements_doc.txt [hunk 6]
```diff
@@ -3,12 +3,12 @@ sphinx!=4.1.0
...
 https://github.com/pydata/pydata-sphinx-theme/archive/f8e179403307ea04ea76940ffcbe6f07a82f0dd8.zip
 https://github.com/sphinx-gallery/sphinx-gallery/archive/master.zip
-sphinxcontrib-bibtex>=2.1.2
+https://github.com/mcmtroffaes/sphinxcontrib-bibtex/archive/4fbda6347a76ad4cc0ea8b62e7bf325e545da291.zip
 memory_profiler
 neo
 seaborn!=0.11.2
 sphinx_copybutton
-sphinx_design
+https://github.com/drammock/sphinx_design/zipball/fix-traverse
```
Comment (drammock) [inline requirements_doc.txt:11]:
```suggestion
https://github.com/drammock/sphinx-design/zipball/fix-traverse
```

**Pull request merged**